tests/playwright/sanity/modules/nested-accordion/helper.ts

```diff
@@ -124,7 +124,7 @@ export async function deleteItemFromRepeater( editor: EditorPage, accordionID: s
 		numberOfContents = await nestedAccordionItemContent.count();
 
 	// Act
-	await deleteItemButton.last().click();
+	await deleteItemButton.nth( 1 ).click();
```

Comment:
I want tam deleting the 2nd item of three, and that I check ithe remaining items id's are updated.